src/main/java/org/broadinstitute/hellbender/tools/sv/PrintSVEvidence.java

```diff
@@ -66,7 +66,7 @@
 )
 @ExperimentalFeature
 @DocumentedFeature
-public final class PrintSVEvidence <F extends Feature> extends FeatureWalker<F> {
+public final class PrintSVEvidence extends FeatureWalker<Feature> {
```

Comment:
Is there any reason to keep this code? I think I would prefer if your new tool took the name `PrintSVEvidence` and we just deleted this.

Reply:
PrintSVEvidence --> dead
MergeSVEvidence --> PrintSVEvidence